docs/detections/alerts-view-details.asciidoc

```diff
@@ -111,8 +111,7 @@ The Investigation section provides the following information:
 +
 TIP: Add an <<add-ig-actions-rule,investigation guide>> to a rule when creating a new custom rule or modifying an existing custom rule's settings.
 
-* **Highlighted fields**: Shows relevant fields for the alert and any custom highlighted fields you added to the rule. 
-//link to custom highlighted fields docs
+* **Highlighted fields**: Shows relevant fields for the alert and any custom highlighted fields you added to the rule. Custom highlighted fields with values are added to this section. Those without values aren't added.
```

Comment:
Did you decide against your code comment `//link to custom highlighted fields docs`? Seems like a link would help establish context, build an EPPO-style web of info. Feel free to ignore if you already decided against it tho.

Reply:
I was going to write a lengthy response outlining the reasons I considered, but you've convinced me 😅 I'll add the link here and in the Serverless docs.

```suggestion
* **Highlighted fields**: Shows relevant fields for the alert and any <<ule-ui-advanced-params,custom highlighted fields>> you added to the rule. Custom highlighted fields with values are added to this section. Those without values aren't added.
```

Reply:
I'm sure there are good arguments either way, but thanks for considering